test/e2e/base_user_test.go

```diff
@@ -15,24 +15,25 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 type baseUserIntegrationTest struct {
 	suite.Suite
-	namespace  string
-	ctx        *framework.Context
-	awaitility *wait.Awaitility
-	hostAwait  *wait.HostAwaitility
+	ctx         *framework.Context
+	hostAwait   *wait.HostAwaitility
+	memberAwait *wait.MemberAwaitility
 }
 
 func (s *baseUserIntegrationTest) clearApprovalPolicyConfig() error {
 	cm := &corev1.ConfigMap{
 		ObjectMeta: v1.ObjectMeta{
-			Name: "toolchain-saas-config",
+			Namespace: s.hostAwait.Namespace,
+			Name:      "toolchain-saas-config",
 		},
 	}
 
-	err := s.awaitility.KubeClient.CoreV1().ConfigMaps(s.namespace).Delete(context.TODO(), cm.Name, v1.DeleteOptions{})
+	err := s.hostAwait.Client.Delete(context.TODO(), cm)
```

Comment:
The truth is, I was not able to see how we could have 2 `Framework` clients (one for the `host` and one for `member`). AFAICS, the `Framework` is a unique, global variable. If there is a way to instantiate another `Framework` client for the `member` awaitility, I'll be happy to use it.


I agree that this may cause issues with the cleaning of test resources. But: 
1. this is already the case with `UserSignups` created via the registration service endpoint
2. resources created on the member clusters should be deleted by the controllers (if all goes well)

Reply:
I wasn't worried about the member cluster cleanup and I saw the code in ToolchainCluster tests that deletes the resources in a "defer" function. I was more worried about the cleanup in host operator... I see the difficulties - keep it as it is and we will see if we can improve it later ;-)